packages/firestore/firestore.rules

```diff
@@ -132,13 +132,14 @@ service cloud.firestore {
             bookingData.date == bookedSlot.data.date &&
             // Check interval integrity
             bookingData.interval in bookedSlot.data.intervals     
+            bookedSlot.
           );
         }
         function checkBookedSlotCategory(slotId) {
           let customer = get(/databases/$(database)/documents/organizations/$(organization)/bookings/$(secretKey));
           let bookedSlot = get(/databases/$(database)/documents/organizations/$(organization)/slots/$(slotId));
           
-          return customer.data.category in bookedSlot.data.categories
+          return bookedSlot.data.categories.hasOnly(customer.data.category)
```

Comment:
Ok, so here I would do `customer.data.category.hasAny(bookedSlot.data.categories)`

It works both ways, but this way it's clear that you're checking customer, not a slot